modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/Archetyper.java

```diff
@@ -78,224 +46,68 @@
  */
 public class Archetyper {
 
-	public ArchetypeGenerationResult generateProject(
-			ProjectTemplatesArgs projectTemplatesArgs, File destinationDir)
-		throws Exception {
-
-		List<File> archetypesDirs = projectTemplatesArgs.getArchetypesDirs();
-		String artifactId = projectTemplatesArgs.getName();
-		String author = projectTemplatesArgs.getAuthor();
-		String className = projectTemplatesArgs.getClassName();
-		boolean dependencyManagementEnabled =
-			projectTemplatesArgs.isDependencyManagementEnabled();
-		String groupId = projectTemplatesArgs.getGroupId();
-		String liferayVersion = projectTemplatesArgs.getLiferayVersion();
-		String packageName = projectTemplatesArgs.getPackageName();
-
-		File templateFile = _getTemplateFile(projectTemplatesArgs);
-
-		String liferayVersions = FileUtil.getManifestProperty(
-			templateFile, "Liferay-Versions");
-
-		if ((liferayVersions != null) &&
-			!_isInVersionRange(liferayVersion, liferayVersions)) {
-
-			throw new IllegalArgumentException(
-				"Specified Liferay version is invalid. Must be in range " +
-					liferayVersions);
-		}
-
-		if (Objects.isNull(groupId)) {
-			groupId = packageName;
-		}
-
-		File workspaceDir = WorkspaceUtil.getWorkspaceDir(destinationDir);
-
-		String projectType = "standalone";
-
-		if (workspaceDir != null) {
-			projectType = WorkspaceUtil.WORKSPACE;
-		}
-
-		String template = projectTemplatesArgs.getTemplate();
-
-		ArchetypeGenerationRequest archetypeGenerationRequest =
-			new ArchetypeGenerationRequest();
-
-		archetypeGenerationRequest.setArchetypeArtifactId(
-			ProjectTemplates.TEMPLATE_BUNDLE_PREFIX +
-				template.replace('-', '.'));
-		archetypeGenerationRequest.setArchetypeGroupId("com.liferay");
-
-		// archetypeVersion is ignored
-
-		archetypeGenerationRequest.setArchetypeVersion("0");
-
-		archetypeGenerationRequest.setArtifactId(artifactId);
-		archetypeGenerationRequest.setGroupId(groupId);
-		archetypeGenerationRequest.setInteractiveMode(false);
-		archetypeGenerationRequest.setOutputDirectory(destinationDir.getPath());
-		archetypeGenerationRequest.setPackage(packageName);
-
-		String buildType = "gradle";
+	public static VelocityComponent createVelocityComponent() throws Exception {
```

Comment:
If the `static` keyword isn't needed, please remove it so its consistent with the other methods in this class.